drivers/shared/executor/executor_linux.go

```diff
@@ -78,17 +79,43 @@ type LibcontainerExecutor struct {
 	userProc       *libcontainer.Process
 	userProcExited chan interface{}
 	exitState      *ProcessState
+	sigChan        chan os.Signal
+}
+
+func (l *LibcontainerExecutor) catchSignals() {
+	l.logger.Trace("waiting for signals")
+	defer signal.Stop(l.sigChan)
+	defer close(l.sigChan)
+
+	signal.Notify(l.sigChan, syscall.SIGHUP, syscall.SIGQUIT, syscall.SIGTERM, syscall.SIGINT, syscall.SIGSEGV)
+	for {
+		signal := <-l.sigChan
+		if signal == syscall.SIGTERM || signal == syscall.SIGKILL || signal == syscall.SIGINT {
```

Comment:
`SIGKILL` isn't one of the signals we've masked above, and we can't catch it anyways. So this block really is only translating `SIGTERM` to `SIGINT`, right?